New sensor: operation, report last and recent operation events

dvx76 opened the pull request:
Add a new sensor, `operation`, which reports the state of the most recent event of type `EventType.OPERATION`. Attributes include additional details like operation name, error code, and a history of the previouys 4 operations.

Screenshot is from an early version which didn't filter the fields to be displayed in the history, but it gives an idea.

Bunch of formatting updates because json files had a mix of tabs and spaces 👀 

![image](https://github.com/user-attachments/assets/f475e4ff-130c-45d1-8d05-3ad8785f26a6)

WebSpider commented on custom_components/myskoda/sensor.py:
```diff
@@ -76,6 +78,53 @@ def _status(self) -> ChargingStatus | None:
+    entity_description = SensorEntityDescription(
+        key="operation",
+        translation_key="operation",
+        device_class=SensorDeviceClass.ENUM,
```
Maybe this should be a diagnostic sensor, and maybe even hidden by default?

dvx76 replied:
Diagnostic definitely - adding that now. Not sure about hidden. This could be useful for many users and it's harder to discover it if it's hidden by default :)

WebSpider reviewed (commented):
Great addition! :muscle: 

One small comment on the entity class we may want to use. 
Oh, and it turns out we have different default settings for jsons :rofl:

WebSpider reviewed (approved):
Otherwise lgtm!

dvx76 commented:
Rebased and added `entity_category=EntityCategory.DIAGNOSTIC`. Ready to merge. @WebSpider let me know if your comment can be resolved. Feel free to just merge.
